demisto_sdk/commands/common/tests/git_util_test.py

```diff
@@ -49,3 +49,22 @@ class Object:
     refs_other.refs = ["a", "b"]
     repo_with_remotes_refs_other.remotes.append(refs_other)
     assert not GitUtil.find_primary_branch(repo_with_remotes_refs_other)
+
+
+def test_git_util_with_repo():
+    """
+    Given
+        - A Git repo.
+
+    When
+        - Creating GitUtil object with git.Repo object.
+
+    Then
+        - Ensure the GitUtil repo path is not None.
+    """
+    from demisto_sdk.commands.common.git_util import GitUtil
+
+    repo = GitUtil.REPO_CLS()
+
+    git_util = GitUtil(repo)
+    assert git_util.repo is not None
```

Comment:
```suggestion
    assert git_util.repo is not None
    assert git_util.repo_path == repo.working_dir
```

Reply:
git_util is not contains repo_path, changed to:
 `git_util.repo.working_dir`